config/datatypes_conf.xml.sample

```diff
@@ -443,8 +443,10 @@
     <sniffer type="galaxy.datatypes.tabular:Vcf"/>
     <sniffer type="galaxy.datatypes.binary:TwoBit"/>
     <sniffer type="galaxy.datatypes.binary:GeminiSQLite"/>
-    <sniffer type="galaxy.datatypes.proteomics:MzSQlite"/>
+    <sniffer type="galaxy.datatypes.binary:MzSQlite"/>
```

Comment:
Changing the sniffer class it's not enough, you should also change the `<datatype>` element above. But probably changed all of this is not a good idea because it will break a lot of stuff for people updating.